### PR TITLE
add a 3rd tile,sync form en version

### PR DESCRIPTION
**Pull request by yeqiugt**
add "ServiceAccount控制器 {#ServiceAccount controller}" to https://kubernetes.io/zh-cn/docs/reference/access-authn-authz/service-accounts-admin/#serviceaccount-admission-controller

### CODE REVIEW

**Comment by yeqiugt** [comment]
Why did the build fail?

File: content/zh-cn/docs/reference/access-authn-authz/service-accounts-admin.md [hunk 1]
```diff
@@ -244,11 +244,15 @@ to obtain short-lived API access tokens is recommended instead.
...
 ## Control plane details
 
+### ServiceAccount controller
+
 A ServiceAccount controller manages the ServiceAccounts inside namespaces, and
 ensures a ServiceAccount named "default" exists in every active namespace.
 -->
 ## 控制平面细节   {#control-plane-details}
 
+### ServiceAccount控制器   {#ServiceAccount controller}
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/service-accounts-admin.md:254]:
```suggestion
### ServiceAccount 控制器   {#serviceaccount-controller} 
```

**Comment by windsonsea** [comment]
Please try to squash your commits into a single one: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by yeqiugt** [comment]
> Please try to squash your commits into a single one: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits
OK, finished

**Review by windsonsea — COMMENTED** [review]
Thanks. This file was synced on Sep 10 in #42977 and no more in-synced content with en upstream.

/approve

**Comment by tengqm** [comment]
Please recommit to trigger the netlify gate.

**Comment by yeqiugt** [comment]
> Please recommit to trigger the netlify gate.

What should I do?

**Comment by tengqm** [comment]
> > Please recommit to trigger the netlify gate.
> 
> What should I do?

In your local console do this:

```
git commit --amend --no-edit --allow-empty
git push -f
```

**Comment by yeqiugt** [comment]
> > > Please recommit to trigger the netlify gate.
> > 
> > 
> > What should I do?
> 
> In your local console do this:
> 
> ```
> git commit --amend --no-edit --allow-empty
> git push -f
> ```

Thanks, finished.

**Pull request merged**